app/views/civil_servants/civil_servants/_login_information_form.html.slim

```diff
@@ -0,0 +1,11 @@
+= simple_form_for(civil_servant, url: civil_servants_civil_servant_path, remote: true) do |f|
+  .modal-body
+    = f.simple_fields_for :user do |user_form|
+      = user_form.input :email
+      = user_form.input :language, collection: User.languages.keys,
+              label_method: ->(label) { t(label, scope: %i[activerecord attributes user languages]) },
+              include_blank: false
```

Comment:
Why is this needed, do we have blank languages?

Reply:
No but the input has by default a blank option. Which we don't want since it would be invalid

Reply:
Ah, makes sense

Reply:
Even if you specify `required: true`

Reply:
Yes